tests/integration/api/setting_apis_test.go

```diff
@@ -3,149 +3,67 @@ package api_test
 import (
 	"fmt"
 	"net/http"
-	"strings"
 
-	"github.com/guonaihong/gout"
 	. "github.com/onsi/ginkgo"
 	"github.com/tidwall/gjson"
-	"github.com/tidwall/sjson"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
 
-	"github.com/rancher/harvester/pkg/config"
-	"github.com/rancher/harvester/pkg/settings"
+	harvesterv1alpha1 "github.com/rancher/harvester/pkg/apis/harvester.cattle.io/v1alpha1"
 	. "github.com/rancher/harvester/tests/framework/dsl"
-	"github.com/rancher/harvester/tests/framework/fuzz"
 	"github.com/rancher/harvester/tests/framework/helper"
 )
 
-var _ = Describe("verify settings apis", func() {
+var _ = Describe("Verify settings' apis", func() {
```

Comment:
change to `verify settings APIs`
So as to keep consistent with the other codes given that in most cases the description starts in lowercase.
Check the other places in this PR as well.